src/main/java/com/minecolonies/network/messages/BuildToolPlaceMessage.java

```diff
@@ -39,6 +40,12 @@
 
     private boolean isHut;
 
+
+    /**
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) First sentence should end with a period. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=checkstyle%3AJavadoc_Style)